package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-add-api-key",
-  "version": "3.2.0",
+  "version": "3.2.1",
```

Comment:
@christopherchubb this should be minor version bump not a patch

Reply:
OK, done.